### DISCUSSION TITLE
Install hassio en Docker

### DISCUSSION THREAD

**Discussion by FR1990** [home-assistant/operating-system#3077] [General]
I want to install hassio + supervisor in docker inside an rpi5
Could someone tell me if there is a manual?
thank you

**Comment by igorpecovnik** [comment]
I used those manuals:
https://www.home-assistant.io/installation/

If you want to skip troubles - Armbian bookworm + Docker + HA supervised made in automated way:
https://github.com/armbian/distribution/releases/download/23.11.2/Armbian_23.11.2_Rpi5b_bookworm_current_6.6.11-homeassistant_minimal.img.xz

**Reply by frenck** [reply]
Please note: Running this on Armbian is not supported by the Home Assistant project. I would not recommend on using the above.

**Reply by igorpecovnik** [reply]
Its not officially supported by Armbian project neither, but that is a matter of will and interest. Biggest problem is anyway securing low level hw stability, which is done by Armbian ... that is specialized in doing that. Application level is running in a container ... I totally understand the value of dedicated OS to run automation and to limit support burden, but many people are asking to have a normal OS (Debian) + HA. This is an answer. Myself, I want to run OS that doesn't give me just HA. Its also an experiment & demonstration of what Armbian (build framework) can do.

**Reply by frenck** [reply]
You are free to do what you want with the software. However, as said, we don't support it. You system will run in an unsupported state (flag), and any issue created with it, will be closed. You are fully on your own.

Therefore, on our issue discussions/support/issue trackers, we have to be sure and make clear when referring people to things like you did above, isn't supported and not recommended (which wasn't the case, hence me responding).

> I want to run OS that doesn't give me just HA

That isn't support for this installation method either. Dedicated hardware is needed. If you want to mix and match, please use a Docker container or install Home Assistant in Python.

**Comment by frenck** [comment]
> I want to install hassio + supervisor in docker inside an rpi5 

RPi5 is not yet in a fully working condition for the Home Assistant. There is currently a preview version available, but it is not recommended for production use yet.

More information can be found here:

- https://www.home-assistant.io/blog/2023/10/17/expected-support-for-raspberry-pi-5/
- https://github.com/home-assistant/operating-system/discussions/2844

> Could someone tell me if there is a manual? thank you

Our manuals can be found here: https://www.home-assistant.io/installation
However, RPi5 is not part of that yet, as per above.

The Home Assistant Container installation method can be used though:

https://www.home-assistant.io/installation/linux#install-home-assistant-container

Please note, to make Home Assistant work on a RPi5 at this moment in time, performance optimizations need to be disabled:

<https://www.home-assistant.io/installation/linux#optimizations>

> hassio + supervisor in docker

Just to clear things up: `hassio` isn't a term used anymore. It is ambiguous in its meaning. Supervisor in Docker is a technically correct thing, but also misleading. Often one thinks it can just be added to an existing Docker installation as an option, which isn't the case.

It is better to refer to our 4 different installation methods, in order of general recommendation and complexity:

1. **Home Assistant Operating System**: Our dedicated Operating System, that contains all.
2. **Home Assistant Container**: Run just Home Assisstant in, for example, your Docker environemnt.
3. **Home Assistant Core**: Run just Home Assistant, directly on Python, in your own venv.
4. **Home Assistant Supervised**: Same as the first, but on your own managed Debian Linux system.

It is considered the most advanced installation method in our available methods, and the least recommended.

Hope this helps a bit 👍 

../Frenck

**Comment by FR1990** [comment]
Home Assistant Supervisor is not possible to install it within (docker) portainer?

**Reply by frenck** [reply]
No, it is not just a container. Supervisor is the orchestrator. It basically is something with a similar function as Portainer. See our installation documentation for the options available.
